cmake/DyninstPlatform.cmake

```diff
@@ -99,6 +103,8 @@ if(${_host_os} STREQUAL "Linux")
       set(DYNINST_PLATFORM x86_64-unknown-linux2.4)
     elseif(${_host_arch} STREQUAL "aarch64")
       set(DYNINST_PLATFORM aarch64-unknown-linux)
+    elseif(${_host_arch} STREQUAL "riscv64")
+      set(DYNINST_PLATFORM riscv64-unknown-linux)
```

Comment:
Same comment as above. This isn't needed since the test suite doesn't have a config for riscv.